source/includes/_claim_fund_api.md

```diff
@@ -0,0 +1,1126 @@
+  if(this.readyState === 4) {
+    console.log(this.responseText);
+  }
+});
+
+xhr.open("GET", "%7B%7Bbase_url%7D%7D/api/claim-fund?partner_trx_id=TRX-20250408-001&send_callback=false");
+xhr.setRequestHeader("Content-Type", "application/json");
+xhr.setRequestHeader("Accept", "application/json");
+xhr.setRequestHeader("x-oy-username", "{{username}}");
+xhr.setRequestHeader("x-api-key", "{{api-key}}");
+
+xhr.send();
+```
+
+```php
+<?php
+require_once 'HTTP/Request2.php';
+$request = new HTTP_Request2();
+$request->setUrl('{{base_url}}/api/claim-fund?partner_trx_id=TRX-20250408-001&send_callback=false');
+$request->setMethod(HTTP_Request2::METHOD_GET);
+$request->setConfig(array(
+  'follow_redirects' => TRUE
+));
+$request->setHeader(array(
+  'Content-Type' => 'application/json',
+  'Accept' => 'application/json',
+  'x-oy-username' => '{{username}}',
+  'x-api-key' => '{{api-key}}'
+));
+try {
+  $response = $request->send();
+  if ($response->getStatus() == 200) {
+    echo $response->getBody();
+  }
+  else {
+    echo 'Unexpected HTTP status: ' . $response->getStatus() . ' ' .
+    $response->getReasonPhrase();
+  }
+}
+catch(HTTP_Request2_Exception $e) {
+  echo 'Error: ' . $e->getMessage();
+}
+```
+
+```python
+import http.client
+import json
+
+conn = http.client.HTTPSConnection("{{base_url}}")
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'x-oy-username': '{{username}}',
+  'x-api-key': '{{api-key}}'
+}
+conn.request("GET", "/api/claim-fund?partner_trx_id=TRX-20250408-001&send_callback=false", headers)
+res = conn.getresponse()
+data = res.read()
+print(data.decode("utf-8"))
+```
+
+> The above command returns JSON structured similar like this:
+
+```json
+{
+  "status": {
+    "code": "000",
+    "message": "Success"
+  },
+  "tx_status": "SUCCESS",
+  "tx_status_description": "The Claim Fund transaction has completed",
+  "amount": 500000,
+  "partner_trx_id": "TRX-20250408-001",
+  "recipient_email": "recipient@mail.com",
+  "recipient_name": "Recipient",
+  "note": "note",
+  "claim_fund_link": "https://claimfund.oyindonesia.com/claim-fund/claim/251280aa-95cb-41f2-84c3-8d2b247aa2b4?token=1ac006ce-82d9-4cb6-9afd-a621ca8dafa0",
+  "expiration": "2025-04-08 12:10:00",
+  "created": "2025-04-08 02:10:00",
+  "updated": "2025-04-08 02:10:00"
+}
+
+```
+
+This endpoint is used to get Claim Fund transaction details and resend callback.
+
+### HTTPS Request
+**[Production]** `GET https://partner.oyindonesia.com/api/claim-fund?partner_trx_id=:partner_trx_id&send_callback=:send_callback`<br>
+**[Staging]** `GET https://api-stg.oyindonesia.com/api/claim-fund?partner_trx_id=:partner_trx_id&send_callback=:send_callback`
+
+### Query Parameters
+
+Parameter | Type | Required | Description
+--------- | ---- | -------- | -----------
+partner_trx_id | String(255) | TRUE | Unique Transaction ID for a specific request, generated by the Client. partner_trx_id should have status = WAITING_APPROVAL.
```

Comment:
descriptionnya belum sesuai. nitip diganti jadi `Unique Transaction ID for a specific request, generated by the Client` @guntoroyk